content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -474,6 +474,56 @@ can lose track of volume labels after restart. This deficiency has been fixed
 in Kubernetes 1.26.
 {{< /note >}}
 
+## Changing the permissions of your's '/proc'
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+By default, containers have multiple paths that are both
+[masked](https://github.com/opencontainers/runtime-spec/blob/f66aad47309/config-linux.md#masked-paths)
```

Comment:
I wouldn't put that link here, although I **would** put it inside the further reading section.